src/ExtendedConv.jl

```diff
@@ -5,8 +5,16 @@
 
 module ExtendedConv
 
-using DrWatson
-import Pkg; Pkg.instantiate()
```

Comment:
This will break `using ExtendedConv` since it will loop trying to instantiate itself, especially with latest julia version

Reply:
Thanks. Will fix in other instances as well.